packages/data-stores/src/domain-suggestions/reducer.ts

```diff
@@ -3,24 +3,71 @@
  */
 import type { Reducer } from 'redux';
 import { combineReducers } from '@wordpress/data';
+import type { TimestampMS } from 'wp-calypso-client/types';
```

Comment:
This would probably cause issues when the package is used in wp-admin. You can inline it if you want `ReturnType< typeof Date.now >`.

Reply:
Specifically, packages pulling from `wp-calypso-client` tend to balloon things and make it less independent/more unpredictable.

Preferably packages would pull code only from other packages, so sometimes it means packaging things from the client, or duplicating if it's not worth the effort.